stan/math/prim/fun/conv_gaus_line.hpp

```diff
@@ -0,0 +1,67 @@
+#ifndef STAN_MATH_PRIM_FUN_CONV_GAUS_LINE
+#define STAN_MATH_PRIM_FUN_CONV_GAUS_LINE
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/prob/normal_cdf.hpp>
+#include <cmath>
+#include <vector>
+
+namespace stan {
+namespace math {
+
+/*
+evaluate the derivative of conv_gaus_line with respect to x
+*/
+double der_conv_gaus_line(double t0, double t1, double a, double b, double x0,
+                          double sig2) {
```

Comment:
Generally we want prim functions to be templated generically to work with any type, then the specialization in `rev` will be for analytic calculations.

Reply:
moved this function to the rev version of this file